android/src/main/java/com/reactnativepagerview/PagerViewViewManager.kt

```diff
@@ -61,49 +58,47 @@ class PagerViewViewManager : ViewGroupManager<ViewPager2>() {
   }
 
   private fun setCurrentItem(view: ViewPager2, selectedTab: Int, scrollSmooth: Boolean) {
-    view.post {
-      view.measure(
-        View.MeasureSpec.makeMeasureSpec(view.width, View.MeasureSpec.EXACTLY),
-        View.MeasureSpec.makeMeasureSpec(view.height, View.MeasureSpec.EXACTLY))
-      view.layout(view.left, view.top, view.right, view.bottom)
-    }
+    view.post { reLayoutView(view) }
     view.setCurrentItem(selectedTab, scrollSmooth)
   }
 
   override fun addView(parent: ViewPager2, child: View, index: Int) {
-    if (child == null) {
-      return
+    val adapter = parent.adapter as FragmentAdapter
+    adapter.addReactView(child, index)
+    parent.post {
```

Comment:
Can we move this code to separate function called `postNewChanges` ? I spotted it is repeated few times.